lib/mayaUsd/ufe/UsdAttribute.cpp

```diff
@@ -81,7 +82,8 @@ template <typename T> bool setUsdAttr(const PXR_NS::UsdAttribute& attr, const T&
     // our own in the StagesSubject, which we invoke here, so that only a
     // single UFE attribute changed notification is generated.
     MayaUsd::ufe::AttributeChangedNotificationGuard guard;
-    return attr.Set<T>(value);
+    bool isSetAttrAllowed = MayaUsd::ufe::isAttributeEditAllowed(attr);
```

Comment:
As per my proposal, here I'd pass in a pointer to string to receive an eventual error message, and displayError() if the return value is false.

Reply:
Addressed in fe8131c1c837ebf7bf5405e7c594df7b5b7b1db4